sources/niUI/src/WidgetListBox.cpp

```diff
@@ -957,6 +962,10 @@ tBool cWidgetListBox::SelectItemByPos(const sVec2f& avAbsPos)
   if (nNewSelection >= GetNumItems())
     return eFalse;
 
+  // ignore selection change when it triggered by the MouseMove event;
```

Comment:
I think the comment should be: "Ignores the selection change when the selection actually didnt change and it was triggered by a MouseMove".

Now the question becomes, why is it triggered at all if the selection didnt change?

Reply:
Another issue I see also, is how does that behave when there's multiple selections? GetSelected() returns only the first select item in that case, it seems it would have unintended consequences in the multi-selection case.